### DISCUSSION TITLE
Have you published a paper using Xanthos?

### DISCUSSION THREAD

**Discussion by crvernon** [JGCRI/xanthos#61] [Show and tell]
Let us know if you are publishing using or discussing Xanthos!

**Comment by FeralFlows** [comment]
Here are some recent papers in which we used Xanthos to do regional and global gridded analysis of historical and future runoff and hydropower production as part of a publication.

> Wild, T.B., Khan, Z., Zhao, M., Suriano, M., Bereslawski, J., Roberts, P., Cassado, J., Gavino Novillo, M., Clarke, L., Hejazi, M., Miralles-Wilhelm, F., Munoz-Castillo, R., Vernon, C., Snyder, A., Yarlagadda, B., Birnbaum, A., Lamontagne, J., White, D., Ojeda-Matos, G. (2021). The Implications of Global Change for the Co-Evolution of Argentina’s Integrated Energy-Water-Land Systems. Earth’s Future. DOI: 10.1029/2020EF001970.

> Wild, T.B., Khan, Z., Clarke, L., Hejazi, M., Bereslawski, J., Suriano, M., Roberts, P., Casado, J., Miralles-Wilhelm, F., Gavino Novillo, M., Munoz-Castillo, R., Moreda, F., Yarlagadda, B., Lamontagne, J., Birnbaum, A. (2021). Integrated Energy-Water-Land Nexus Planning in the Colorado River Basin (Argentina). Regional Environmental Change. DOI: 10.1007/s10113-021-01775-1.

> Da Silva, S.R.S., Hejazi, M., Iyer, G., Wild, T.B., Binsted, M., Miralles-Wilhelm, F., Patel, P., Snyder, A., Vernon, C. (2021). Power Sector Investment Implications of Climate Impacts on Renewable Resources in Latin America and the Caribbean. Nature Communications.

> Khan, Z., Wild, T.B., Carrazzone, M., Gaudioso, R., Mascari, M.P., Bianchi, F., Weinstein, F., Perez, F., Perez, W., Miralles-Wilhelm, F., Clarke, L., Hejazi, M., Vernon, C., Kyle, P., Edmonds, J., Munoz-Castillo, R., (2020). Integrated energy-water-land nexus planning to guide national policy: An example from Uruguay. Environmental Research Letters. https://doi.org/10.1088/1748-9326/ab9389.
